app/config/Binders.scala

```diff
@@ -29,4 +31,17 @@ object Binders {
 
     override def unbind(key: String, value: Srn): String = value.value
   }
+
+  implicit val max10PathBinder: PathBindable[Max10] =
```

Comment:
This can be made more generic to handle any Refined[Int, ?], just need an instance of Validate[Int, ?] in implicit scope
```
  implicit def refinedIntPathBinder[T](implicit ev: Validate[Int, T]): PathBindable[Refined[Int, T]] =
    new PathBindable[Refined[Int, T]] {
      override def bind(key: String, value: String): Either[String, Refined[Int, T]] = value
        .toIntOption
        .toRight(s"value for key $key was not an Integer")
        .flatMap(refineV[T](_))

      override def unbind(key: String, value: Refined[Int, T]): String = value.value.toString
    }

  implicit def refinedIntJSLiteral[T]: JavascriptLiteral[Refined[Int, T]] =
    (value: Refined[Int, T]) => value.value.toString
```

Reply:
I'm not sure if this is feasible - the Validate instance would look something like this
```
  implicit def max10Validator: Validate[Int, OneToTen] = Validate.instance(
    i => Result.fromBoolean(refineV[OneToTen](i).isRight, i),
    _.toString
  )
```
but since we use refineV (which needs a validator for OneToTen), it will use itself as the implicit instance and cause a StackOverflow - one way to get around it is to not use refine and do a boolean check on the int (i > 0 && <= 10) but thats not using the Refined which doesn't seem right